Jenkinsfile-release

```
@@ -50,6 +50,17 @@ pipeline {
             }
         }
 
+        stage('Commit version for documentation') {
+            steps {
+                script {
+                    sh sed -i "s/^:ispn_version:.*$/:ispn_version: ${version}/" documentation/src/main/asciidoc/topics/attributes/community-attributes.adoc
```

I think you need quotes in the command:
```suggestion
                    sh "sed -i \"s/^:ispn_version:.*$/:ispn_version: ${version}/\" documentation/src/main/asciidoc/topics/attributes/community-attributes.adoc"
```